Code/GraphMol/RGroupDecomposition/RGroupFingerprintScore.cpp

```diff
@@ -24,6 +25,21 @@
 
 namespace RDKit {
 
+namespace detail {
+RData dummyFingerprint(int label) {
```

Comment:
Perhaps a bit more clarity here that a dummy FP is an fp equivalent to hydrogen

Reply:
I've renamed the function and added a comment